src/models/jira-client.ts

```diff
@@ -9,7 +9,7 @@ import { envVars } from "config/env";
 export class JiraClient {
 	axios: AxiosInstance;
 
-	static async getNewClient(installation: Installation, log: Logger) {
+	static async getNewClient(this: void, installation: Installation, log: Logger) {
```

Comment:
Curious how does this work? What will happen with the current calls (which passes 2 args only)?

Reply:
You don't need to update the calls.  You are just telling the class that 'this' needs to be treated as void in the scope of the function